src/constants/index.js

```diff
@@ -1 +1,2 @@
 export * from "./hyperlinks";
+export * from "./socials.js"
```

Comment:
Similar to the above we don't need to add in a file extension, it's resolved automatically